graph/build.gradle

```diff
@@ -1,23 +0,0 @@
-// Apply core plugins.
```

Comment:
Please remove it from `settings.gradle`, too.

Reply:
Done.